arm/Microsoft.KeyVault/vaults/deploy.bicep

```
@@ -179,22 +179,22 @@ resource keyVault 'Microsoft.KeyVault/vaults@2019-09-01' = {
     softDeleteRetentionInDays: softDeleteRetentionInDays
     enableRbacAuthorization: enableRbacAuthorization
     createMode: createMode
-    enablePurgeProtection: ((!enablePurgeProtection) ? null : enablePurgeProtection)
+    enablePurgeProtection: enablePurgeProtection ? enablePurgeProtection : null
```

```suggestion
    enablePurgeProtection: enablePurgeProtection
```
Is this not a boolean parameter with a default value of `false`? Why are we setting it to null?

---

@MariusStorhaug we actually had an issue for that (https://github.com/Azure/ResourceModules/issues/1038#issuecomment-1057134138). In essence, that service only accepts `true` or nothing.

---

:D

---

Should we raise that to the PG?